docs/source/ko/_toctree.yml

```
@@ -293,8 +293,8 @@
     - local: in_translation
       title: (번역중) Optimization
     - local: in_translation
-      title: (번역중) Model outputs
-    - local: in_translation
+      title: main_classes/Model outputs
+    - local: 모델 출력
```

I think this should be reversed.

```suggestion
      title: 모델 출력
    - local: main_classes/output
```

---

Thanks for pointing this out!